metals/src/main/scala/scala/meta/internal/metals/Compilations.scala

```diff
@@ -265,7 +265,7 @@ final class Compilations(
       userConfiguration().verboseCompilation && (connection.isBloop || connection.isScalaCLI)
     ) {
       params.setArguments(List("--verbose", "--best-effort").asJava)
-    } else { params.setArguments(List("--best-effort").asJava) }
+    } else if (!connection.isBazel) { params.setArguments(List("--best-effort").asJava) }
```

Comment:
```suggestion
    } else if (connection.isBloop || connection.isScalaCLI) { params.setArguments(List("--best-effort").asJava) } else  params.setArguments(Nil.asJava)
```
I mixed up the condition, it should be this. I also saw some issue somewhere when arguments were null. I think that was bazel BSP